### PR TITLE
Fix: 0 considered as false

### PR DESCRIPTION
**Pull request by 0xcodercrane**
Resolves #https://github.com/ubiquity/ubiquibot/issues/404

### CODE REVIEW

**Comment by 0xcodercrane** [comment]
QA: https://github.com/ubiquibot/staging/issues/31

**Review by pavlovcik — APPROVED** [review]

**Review by pavlovcik — CHANGES_REQUESTED** [review]
https://github.com/ubiquibot/staging/issues/31#issuecomment-1596194441

File: src/handlers/payout/action.ts [hunk 1]
```diff
@@ -48,8 +48,8 @@ export const handleIssueClosed = async () => {
   const multiplier = await getWalletMultiplier(assignee.login);
 
   if (multiplier === 0) {
-    logger.info(`Skipping to proceed the payment because multiplier is 0`);
-    return "Skipping to proceed the payment because multiplier is 0";
+    logger.info(`Refusing to generate the payment permit because @${assignee.login} payment multiplier is 0`);
+    return `Refusing to generate the payment permit because @${assignee.login} payment multiplier is 0`;
```
[the remark this answers is not in the document]
Comment (0xcodercrane) [reply src/handlers/payout/action.ts:52]: `Why not use a string variable and then use it twice?
`
we have similar things in several places. so I'd like to process them at once

[the remark this answers is not in the document]
Comment (pavlovcik) [reply src/handlers/payout/action.ts:52]: Will you be able to address the changes I requested?

[the remark this answers is not in the document]
Comment (0xcodercrane) [reply src/handlers/payout/action.ts:52]: > Will you be able to address the changes I requested?

fixed it here. https://github.com/ubiquity/ubiquibot/commit/c5d916f079d12d0f31040b68b90b77754a00f0e0

**Pull request merged**